test/date_range_test.rb

```diff
@@ -35,32 +35,32 @@
       expect(@range.overlap?(test_range)).must_equal true
     end
 
-    xit "returns true for a contained range" do
+    it "returns true for a contained range" do
     end
 
-    xit "returns true for a range that overlaps in front" do
+    it "returns true for a range that overlaps in front" do
     end
 
-    xit "returns true for a range that overlaps in the back" do
+    it "returns true for a range that overlaps in the back" do
     end
 
-    xit "returns true for a containing range" do
+    it "returns true for a containing range" do
     end
 
-    xit "returns false for a range starting on the end_date date" do
+    it "returns false for a range starting on the end_date date" do
     end
 
-    xit "returns false for a range ending on the start_date date" do
+    it "returns false for a range ending on the start_date date" do
     end
 
-    xit "returns false for a range completely before" do
+    it "returns false for a range completely before" do
     end
 
-    xit "returns false for a date completely after" do
+    it "returns false for a date completely after" do
     end
   end
 
-  xdescribe "include?" do
+  describe "include?" do
```

Comment:
The tests below should be completed. This will bring up you test coverage `for DateRange`